relation_engine_server/utils/json_validation.py

```diff
@@ -10,10 +10,17 @@
     Validator(schema).validate(obj)
```

Comment:
I have a feeling that down the line we will want to make this into a pip package so we can use it in other repos